### DISCUSSION TITLE
Can't install LuCI and passwall on latest snapshot

### DISCUSSION THREAD

**Discussion by Mohsen7s** [immortalwrt/immortalwrt#1392] [Q&A] [answered]
### Describe the bug

Currently there is no version released to support orange pi 5 plus so I have to use the snapshot. However when I try to install LuCI Im getting errors.
I cant either install passwall. Here is the log

### ImmortalWrt version

Snapshot

### ImmortalWrt release

Snapshot

### ImmortalWrt target/subtarget

RK3588

### Device

Orange Pi 5 Plus

### Image kind

Official downloaded image

### Steps to reproduce

opkg update
opkg install luci

### Actual behaviour

Can't find dependency kernel 6.6

### Expected behaviour

Should find dependency and install regulary

### Additional info

root@ImmortalWrt:~# opkg install luci-app-passwall
Installing luci-app-passwall (4.78-1) to root...
Downloading https://downloads.immortalwrt.org/snapshots/packages/aarch64_generic/luci/luci-app-passwall_4.78-1_all.ipk
Collected errors:
 * pkg_hash_check_unresolved: cannot find dependency kernel (= 6.6.39~d8d97f68b06125f9b2a13d44b337b50f-r1) for kmod-nf-reject
 * pkg_hash_check_unresolved: cannot find dependency kernel (= 6.6.39~d8d97f68b06125f9b2a13d44b337b50f-r1) for kmod-nf-reject6
 * pkg_hash_check_unresolved: cannot find dependency kernel (= 6.6.39~d8d97f68b06125f9b2a13d44b337b50f-r1) for kmod-nf-log
 * pkg_hash_check_unresolved: cannot find dependency kernel (= 6.6.39~d8d97f68b06125f9b2a13d44b337b50f-r1) for kmod-nf-log6
 * pkg_hash_check_unresolved: cannot find dependency kernel (= 6.6.39~d8d97f68b06125f9b2a13d44b337b50f-r1) for kmod-crypto-crc32c
 * pkg_hash_check_unresolved: cannot find dependency kernel (= 6.6.39~d8d97f68b06125f9b2a13d44b337b50f-r1) for kmod-nft-core
 * pkg_hash_check_unresolved: cannot find dependency kernel (= 6.6.39~d8d97f68b06125f9b2a13d44b337b50f-r1) for kmod-nft-socket
 * pkg_hash_check_unresolved: cannot find dependency kernel (= 6.6.39~d8d97f68b06125f9b2a13d44b337b50f-r1) for kmod-nft-tproxy
 * pkg_hash_check_unresolved: cannot find dependency kernel (= 6.6.39~d8d97f68b06125f9b2a13d44b337b50f-r1) for kmod-inet-diag
 * pkg_hash_check_unresolved: cannot find dependency kernel (= 6.6.39~d8d97f68b06125f9b2a13d44b337b50f-r1) for kmod-netlink-diag
 * pkg_hash_check_unresolved: cannot find dependency kernel (= 6.6.39~d8d97f68b06125f9b2a13d44b337b50f-r1) for kmod-tun
 * satisfy_dependencies_for: Cannot satisfy the following dependencies for luci-app-passwall:
 *      kernel (= 6.6.39~d8d97f68b06125f9b2a13d44b337b50f-r1)
 * opkg_install_cmd: Cannot install package luci-app-passwall.

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by 1715173329** [comment, marked as answer]
snapshot feeds are currently broken due to recent DDoS attack. it can take up to 2 weeks to recover.
